pyvecorg/templates/index.html

```diff
@@ -237,10 +237,13 @@ <h3>{{ meta.bank_account.heading }}</h3>
                         <h3>{{ meta.downloads.heading }}</h3>
                         <p class="downloads">
                             <i class="fa fa-file-pdf-o" aria-hidden="true"></i>
-                            <a href="{{ url_for('static', filename='bylaws/stanovy.pdf') }}">{{ meta.downloads.bylaws }}.pdf</a>
+                            <a href="{{ url_for('static', filename='bylaws.pdf') }}">{{ meta.downloads.bylaws }}.pdf</a>
 
-                            <i class="fa fa-file-text-o" aria-hidden="true"></i>
-                            <a href="{{ url_for('static', filename='bylaws/stanovy.rtf') }}">{{ meta.downloads.bylaws }}.rtf</a>
+                            <i class="fa fa-file-o" aria-hidden="true"></i>
+                            <a href="https://docs.pyvec.org/operations/bylaws.html">{{ meta.downloads.bylaws }}</a>
+
+                            <i class="fa fa-archive" aria-hidden="true"></i>
+                            <a href="https://or.justice.cz/ias/ui/vypis-sl-firma?subjektId=760829">{{ meta.downloads.filing }}</a>
```

Comment:
This is only accessible through justice.cz, and unfortunately there is no English alternative.